packages/app/src/app/overmind/namespaces/dashboard/state.ts

```diff
@@ -71,6 +74,8 @@ export type State = {
     older: Sandbox[];
   };
   curatedAlbums: DashboardAlbum[];
+  // TODO: fix unknown
+  contributions: DashboardContributionBranch[] | null;
```

Comment:
@alexnm I was a bit conflicted by this because these branches are not really sandboxes, right? I like keeping lists under the same structure but the types seem conceptually wrong, and we also have lists under the general structure such as the `curatedAlbums`, so the contributions lists are not the only thing and don't seem too out of place. Wdyt?

Reply:
maybe the word sandboxes there is misleading, but indeed, let's see how this works in the end. for the recent page, I'm adding the branches under `RECENT_BRANCHES` inside sandboxes as it gets merged with `RECENT_SANDBOXES`